src/components/NoteWhiteArea.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { note } from './NoteWhiteArea.scss';
+
+const NoteWhiteArea = ({ className, children, ...props }) =>
```

Comment:
What is the purpose of this component? Right now it's a `div` with a default class, and a lot of options. Can we remove the `classname` and `props` ?